core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -712,6 +714,9 @@ object KafkaConfig {
   val MetadataLogDirDoc = "This configuration determines where we put the metadata log for clusters in KRaft mode. " +
     "If it is not set, the metadata log is placed in the first log directory from log.dirs."
   val MetadataSnapshotMaxNewRecordBytesDoc = "This is the maximum number of bytes in the log between the latest snapshot and the high-watermark needed before generating a new snapshot."
+  val MetadataMaxIdleIntervalMsDoc = "This configuration controls how often the active " +
```

Comment:
Can we disallow negative values for this? 0 for disabled seems clear enough.

Reply:
Done.